addons/remix-tree-button/main.js

```diff
@@ -0,0 +1,44 @@
+export default async function ({ addon, global, console }) {
+  //define remix tree button elements
+
+  const subactions = document.querySelector(".flex-row.subactions").querySelector(".action-buttons");
+
+  const remixtree = document.createElement("button");
+  remixtree.className = "button action-button remixtree-button";
+  remixtree.id = "scratchAddonsRemixTreeBtn";
+  remixtree.innerHTML = `
+    <img src="https://scratch.mit.edu/svgs/project/remix-white.svg" height="15px" style="margin-right: 5px;" />
+    <span>Remix Tree</span>
+    `;
+  remixtree.onclick = () => {
+    if (window.location.href.endsWith("/")) window.location.href += "remixtree";
+    else window.location.href += "/remixtree";
+  };
+  remixtree.style.display = "flex";
+  remixtree.style.alignItems = "center";
+  if (addon.settings.get("buttonColor")) {
+    remixtree.style.backgroundColor = hexTest(addon.settings.get("buttonColor"));
```

Comment:
There's no need to check whether it's valid hex. Scratch Addons does that for you

Reply:
ok

Reply:
fixed